Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -1081,7 +1081,7 @@ void VideoConfigDiag::PopulatePostProcessingShaders()
   if (shaders.empty())
     return;
 
-  choice_ppshader->AppendString(_("(off)"));
+  choice_ppshader->AppendString(_("Off"));
```

Comment:
I wasn't sure why this option had parentheses around it as that doesn't match the style of the interface elsewhere. If there was a reason for doing it it can be changed back.